packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -320,6 +320,40 @@ router.addRoute('route-http', {
 });
 ```
 
+Add an http2 route with retries:
+
+```ts
+router.addRoute('route-http2-retry', {
+  routeSpec: appmesh.RouteSpec.http2({
+    weightedTargets: [{ virtualNode: node }],
+    retryPolicy: {
+      httpRetryEvents: [appmesh.HttpRetryEvent.CLIENT_ERROR],
+      tcpRetryEvents: [appmesh.TcpRetryEvent.CONNECTION_ERROR],
```

Comment:
I'm a little confused. Can an `http2` Route specify TCP retry events? Shouldn't an HTTP Route only retry on HTTP events?

Reply:
@skinny85 Unless I'm misunderstanding the docs, yes, HTTP routes can specify TCP retry events: 

- https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-route-httpretrypolicy.html#cfn-appmesh-route-httpretrypolicy-tcpretryevents
- https://docs.aws.amazon.com/app-mesh/latest/APIReference/API_HttpRetryPolicy.html

It seems that the service team has split the specification of retry events by the network stack layer on which the error may occur. The CFN and API docs don't have much to say beyond, "Specify at least one value for at least one of the types of RetryEvents", so I'm not sure if the service team might need to intervene for clarification on this point.

Reply:
I've tweaked the readme example so that the order of these event properties is shown lowest-layer to highest. I hope that if the customer sees TCP connection errors before HTTP, they may realize that they need to think of a retry policy based on events that can occur at any network layer.